spec/spec_helper.rb

```diff
@@ -17,6 +17,13 @@
     server_version = ActiveRecord::Base.connection.select_value("SHOW server_version")
     puts "DEBUG: Connecting to Postgres server version #{server_version}"
 
+    ActiveRecord::Base.connection.execute("DROP EXTENSION IF EXISTS pg_partman CASCADE")
+
+    # Drop parent partition tables first to automatically drop children
+    ActiveRecord::Base.connection.select_values("SELECT c.relname FROM pg_class c JOIN pg_partitioned_table p on c.oid = p.partrelid").each do |table|
+      ActiveRecord::Base.connection.execute("DROP TABLE #{table} CASCADE")
+    end
+
```

Comment:
All of this code isn't _strictly_ necessary yet as we haven't added any tests that create the partman extension / child partitions. We can definitely remove it from this PR if folks feel strongly, but it shouldn't affect existing tests in any way